main.js

```diff
@@ -1,58 +1,45 @@
-let newElement  = document.createElement("h4");
-newElement.textContent = "System Online: Course trajectory activated."
+let newElement = document.createElement("h4");
+newElement.textContent = "System Online: Course trajectory activated.";
 newElement.style.zIndex = "1";
 newElement.style.textAlign = "center";
 newElement.style.color = "white";
 
-let newElement2 =  document.createElement("h4");
+let newElement2 = document.createElement("h4");
 newElement2.textContent = "System Offline: Off Course.";
 newElement2.style.zIndex = "1";
 newElement2.style.textAlign = "center";
 newElement2.style.color = "white";
 
-window.onload=function()
-{
-
-
-
 //     // only can be added once page is loaded
 // const newElement=document.createElement("p");
 // newElement.textContent ="Hello from Javascript!";
 // document.body.appendChild(newElement);
 
-if (this.navigator.onLine)
-{
-  this.console.log("online");
-   onFunction();
-
-}
+window.onload = function() {
+  onFunction();
```

Comment:
This function needs to be called after the page is loaded and the DOM is parsed and ready to be edited, so that's why it goes inside this `window.onload` function.